scripts/zkai_seq32_derived_d128_mlp_surface_gate.py

```
@@ -0,0 +1,1027 @@
+            "input_min_q8": min(values),
+            "input_max_q8": max(values),
+            "input_sum_q8": sum(values),
+        },
+        "non_claims": list(NON_CLAIMS),
+        "validation_commands": list(VALIDATION_COMMANDS),
+    }
+    payload["payload_commitment"] = payload_commitment(payload)
+    return payload
+
+
+def _add_bridge_anchor(module: Any, rmsnorm_payload: dict[str, Any]) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `module`

(ANN401)